### DISCUSSION TITLE
Is Retraining existing model possible?

### DISCUSSION THREAD

**Discussion by pranaycv** [open-edge-platform/anomalib#2241] [General]
Is it possible to retain existing model that is saved with only the new good images, so that the model learn new variation in the object intended to be inspected.

**Comment by alexriedel1** [comment]
It depends I would say! Embedding-similarity-based models wouldn't profit much from out of distribution training examples I guess. Reconstruction-based models could show improvement but thats mostly an assumption.
Either way, i guess retraining existing models wouldn't help you a lot
